src/utils/operators.ts

```diff
@@ -11,7 +11,7 @@ const expToVal = (expression: string | number | object, item: any) => {
   }
 };
 
-const checkAndComputeNestedExpression = (arg: any, item: any): any => {
+export const checkAndComputeNestedExpression = (arg: any, item: any): any => {
```

Comment:
I guess we should add a TypeScript Type here for operators, like we did for accumulators.